tests/tests/test_compat.py

```diff
@@ -0,0 +1,295 @@
+# Copyright 2020 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import pytest
+import time
+
+from datetime import datetime, timedelta, timezone
+
+from testutils.util.artifact import Artifact
+from testutils.infra.container_manager import factory
+from testutils.infra.device import MenderDevice
+from testutils.common import create_org, create_user
+from testutils.api.client import ApiClient
+from testutils.api import deviceauth, useradm, inventory, deployments
+
+from .. import conftest
+
+container_factory = factory.get_factory()
+
+TIMEOUT = timedelta(minutes=5)
+
+
+@pytest.fixture(scope="function")
+def setup_os_compat(request):
+    env = container_factory.getCompatibilitySetup()
+    request.addfinalizer(env.teardown)
+    env.setup()
+
+    env.user = create_user(
+        "test@mender.io", "correcthorse", containers_namespace=env.name
+    )
+    env.populate_clients()
+
+    clients = env.get_mender_clients()
+    env.devices = []
+    for client in clients:
+        dev = MenderDevice(client)
+        dev.ssh_is_opened()
+        env.devices.append(dev)
```

Comment:
As the code is now, these are unused. It only makes sense to have it if we have client-side checks for the test.